src/util/init_ini_data.cpp

```diff
@@ -39,8 +36,12 @@ namespace hpx { namespace util
     bool handle_ini_file (section& ini, std::string const& loc)
     {
         try {
-            namespace fs = boost::filesystem;
+            namespace fs = filesystem;
+#if defined(HPX_HAVE_CXX17_FILESYSTEM)
+            std::error_code ec;
+#else
             boost::system::error_code ec;
+#endif
```

Comment:
Yah, this confirms it. Let's introduce `hpx::filesystem::error_code` instead.

Reply:
Good idea. Thanks!